### DISCUSSION TITLE
Model changes color with auto-rotate enabled

### DISCUSSION THREAD

**Discussion by joaoibsilva** [google/model-viewer#2012] [Q&A]
Hi, i have noticed that when auto-rotate is enabled when interacting with the model the same keeps the lighting of that particular rotation position instead of the initial lighting settings. Please have a look at the image provided.
![erro](https://user-images.githubusercontent.com/12135134/98728187-0bcac400-2391-11eb-9cc9-054be07f371b.jpg)

**Comment by elalish** [comment]
@joaoibsilva True, but this is intentional. It will be more clear why if you set a `skybox-image`. I have heard complaints about the backside of the default environment being too dim. Is this what you would like changed, or do you have a different feature request?

**Comment by mrwobttom** [comment]
I would love to see an option to lock the position of the environment/skybox. Ability to define starting position of the environment/skybox would be great as well. 

I have a lot of models with reflective screens, and the default position of the lighting causes a lot of glare. Once it locks to the rotated image it gets better:
default load:
![image](https://user-images.githubusercontent.com/70220869/107816906-3f1f4500-6d2a-11eb-8d34-6a715fa4cc85.png)

spinning it back, after rotation has started (because the position of the lights is now locked to the side where they were when I rotated):
![image](https://user-images.githubusercontent.com/70220869/107816942-49d9da00-6d2a-11eb-99a5-21c54b71a089.png)

**Comment by elalish** [comment]
@mrwobttom No problem, just use `resetTurntableRotation(theta)`: https://modelviewer.dev/docs/#entrydocs-stagingandcameras-methods-resetTurntableRotation. The environment is in fact locked, it's the model that rotates when using `auto-rotate`. You may also be interested in our more even neutral lighting: https://modelviewer.dev/examples/lightingandenv/#neutralLighting

**Reply by mrwobttom** [reply]
Thanks! I tried environment-image="neutral" and don't see much difference:
![image](https://user-images.githubusercontent.com/70220869/107819923-cbcc0200-6d2e-11eb-892c-dd11c1128b6c.png)

I'm thinking maybe I just need a different hdri environment image
